src/test/resources/suite-testng1.xml

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
```

Comment:
Add also a directory with your name inside resources and move this .xml file in that directory. Thanks!